### PR TITLE
[BOBAX-42] Install Fast API

### PR DESCRIPTION
**Pull request by SimonMTan**
-Install Pip Freeze Requirement (pip freeze > requirements.txt)
-Set up Fast API
-Have one get route working

### CODE REVIEW

**Review by seintun — CHANGES_REQUESTED** [review]
Rebase develop

**Comment by SimonMTan** [comment]
what did you mean by rebase

**Comment by seintun** [comment]
> what did you mean by rebase

Check your commit history. Your branch has old commits that were already merged two weeks ago.
You need to merge the latest from the develop branch

image

File: src/assets/sampleBobaAPI.tsx
Comment (seintun) [inline src/assets/sampleBobaAPI.tsx:1]: Old commits from 2 weeks ago.
Please ensure you're always rebasing your branch with target `develop` branch

File: src/pages/Home/index.tsx
Comment (seintun) [inline src/pages/Home/index.tsx:1]: Old commits from 2 weeks ago.
Please ensure you're always rebasing your branch with target `develop` branch

File: src/types/common/main.d.ts
Comment (seintun) [inline src/types/common/main.d.ts:1]: Old commits from 2 weeks ago.
Please ensure you're always rebasing your branch with target `develop` branch

**Review by seintun — CHANGES_REQUESTED** [review]

**Review by seintun — APPROVED** [review]

**Pull request merged**